src/test/java/ru/practicum/shareit/booking/BookingRepositoryTest.java

```diff
@@ -0,0 +1,416 @@
+                .build();
+
+        item1 = Item.builder()
+                .id(1L)
+                .name("item1 name")
+                .description("seaRch1 description ")
+                .available(true)
+                .owner(user1)
+                .build();
+
+        dateTime = LocalDateTime.of(2023,1,1,10,0,0);
+
+        final int from = Integer.parseInt(UserController.PAGE_DEFAULT_FROM);
+        final int size = Integer.parseInt(UserController.PAGE_DEFAULT_SIZE);
+        pageable = PageRequest.of(from / size, size);
+
+        bookingPast = Booking.builder()
+                .id(1L)
+                .start(dateTime.minusYears(10))
+                .end(dateTime.minusYears(9))
+                .item(item1)
+                .booker(user2)
+                .status(Status.APPROVED)
+                .build();
+
+        bookingCurrent = Booking.builder()
+                .id(2L)
+                .start(dateTime.minusYears(5))
+                .end(dateTime.plusYears(5))
+                .item(item1)
+                .booker(user2)
+                .status(Status.APPROVED)
+                .build();
+
+        bookingFuture = Booking.builder()
+                .id(3L)
+                .start(dateTime.plusYears(8))
+                .end(dateTime.plusYears(9))
+                .item(item1)
+                .booker(user2)
+                .status(Status.WAITING)
+                .build();
+
+        bookingRejected = Booking.builder()
+                .id(4L)
+                .start(dateTime.plusYears(9))
+                .end(dateTime.plusYears(10))
+                .item(item1)
+                .booker(user2)
+                .status(Status.REJECTED)
+                .build();
+    }
+
+    @BeforeEach
+    public void beforeEach() {
+        userRepository.save(user1);
+        userRepository.save(user2);
+        itemRepository.save(item1);
+        bookingRepository.save(bookingPast);
+        bookingRepository.save(bookingCurrent);
+        bookingRepository.save(bookingFuture);
+        bookingRepository.save(bookingRejected);
+    }
+
+    @Nested
+    class FindByBookerIdOrderByStartDesc {
+        @Test
+        public void shouldGetAll() {
+            List<Booking> result = bookingRepository.findByBookerIdOrderByStartDesc(user2.getId(), pageable)
+                    .get().collect(Collectors.toList());
+
+            assertEquals(4, result.size());
+            assertEquals(bookingRejected.getId(), result.get(0).getId());
+            assertEquals(bookingFuture.getId(), result.get(1).getId());
+            assertEquals(bookingCurrent.getId(), result.get(2).getId());
+            assertEquals(bookingPast.getId(), result.get(3).getId());
+        }
+
+        @Test
+        public void shouldGetEmpty() {
+            List<Booking> result = bookingRepository.findByBookerIdOrderByStartDesc(user1.getId(), pageable)
+                    .get().collect(Collectors.toList());
+
+            assertEquals(0, result.size());
```

Comment:
```suggestion
            assertTrue(result.isEmpty());
```
Поправь, пожалуйста, во всех случаях, где проверяется, что список пуст

Reply:
Переделал